hs/database/JsonParser.hs

```diff
@@ -144,13 +145,20 @@ instance FromJSON Tutorial where
         | V.length v == 2 = do
             times <- parseJSON $ v V.! 0
             timeStr <- parseJSON $ v V.! 1
-            return $ Tutorial times timeStr
+            return $ Tutorial Nothing times timeStr
+        | V.length v == 3 = do
+            tutorialSection <- parseJSON $ v V.! 0
+            times <- parseJSON $ v V.! 1
+            timeStr <- parseJSON $ v V.! 2
+            return $ Tutorial tutorialSection times timeStr
         | otherwise = mzero
     parseJSON _ = mzero
 
 instance ToJSON Tutorial where
-  toJSON (Tutorial times timeStr) 
-          = Array (V.fromList (map toJSON times) V.++ (V.singleton $ toJSON timeStr))
+  toJSON (Tutorial tutorialSection times timeStr) 
+          = Array ((V.singleton (case tutorialSection of
+                        Just value -> toJSON value
+                        Nothing -> toJSON (B.unpack "N/A"))) V.++ V.singleton (toJSON (map toJSON times)) V.++ (V.singleton $ toJSON timeStr))
```

Comment:
I am unsure about what the best way to handle this case is. What are your thoughts, @david-yz-liu ?